src/i18n/en-US/systemProfile.ts

```diff
@@ -298,15 +298,22 @@ const systemProfile = {
     }
   },
   systemTable: {
-    title: 'All systems',
-    subtitle: 'Bookmark systems that you want to access more quickly.',
+    title: 'All CMS systems',
+    subtitle:
+      'Click the bookmark icon (<icon />) to bookmark systems that you want to access more quickly.',
```

Comment:
Does this html work in a string like this?

Reply:
Yes with the `<Trans/>` component

https://github.com/CMSgov/easi-app/blob/85343e810b3834479a443065f03e543ddf7097de/src/views/SystemList/index.tsx#L124-L28

Reply:
Oh nice! Cool, thanks 😄